### ISSUE TITLE
Evaluation Metrics

### ISSUE THREAD

**Issue by skill-diver** [naver/mast3r#97]
Hi Naver team, thank you for the great work.
I have a question regarding to evaluation metrics. I am wondering where can I find the same evaluation metric code as your paper show? Since in your training code I just find the metrics of different matching loss.

**Comment by skill-diver** [comment]
I know the matching metrics in your code named "matching loss" is for 3D construction. Do you have some more reliable matching metrics only for feature matching?

**Comment by Linusnie** [comment]
I am wondering this as well. Are evaluation scripts for the paper tables available anywhere? If not, any plans to release them?

**Comment by andrewyguo** [comment]
Following this issue as well

**Comment by soohyoen** [comment]
@skill-diver 

I'm experiencing the same problem. Were you able to resolve this?
